src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -275,7 +283,6 @@ public function payment_fields() {
 	 */
 	public function should_display_plan( $plan_key, $gateway_id ) {
 		$alma_settings  = alma_wc_plugin()->settings;
-		$should_display = false;
```

Comment:
I wouldn't have remove this line.

Reply:
according to following method behavior... `$should_display` is always set... first instanciation is redundant then not necessary.